packages/ts-migrate-plugins/src/plugins/hoist-class-statics.ts

```diff
@@ -90,11 +90,10 @@ function isAlreadyHoisted(
 }
 
 function hoistStaticClassProperties(
-  fileName: string,
+  sourceFile: ts.SourceFile,
   sourceText: string,
   options: Options,
 ): string {
-  const sourceFile = ts.createSourceFile(fileName, sourceText, ts.ScriptTarget.Latest, true);
```

Comment:
I also noticed we were creating a new source file here when we could have passed through the one given to `run`, and couldn't see a reason for doing that.